src/__tests__/api.test.ts

```diff
@@ -157,4 +161,62 @@ describe("flashcard", () => {
       })
     })
   })
+  describe("deleting card", () => {
+    describe("delete card route", () => {
+      it("function deletes the requested flashcard if it was created less than 5 minutes ago and returns a status code of 204", async () => {
+        const idCardToDelete = new ObjectId(Date.now() - 5000).toString()
+        flashcardModel.create({
+          ...newCardMock,
+          createdAt: Date.now() - 5000,
+          _id: idCardToDelete,
+        })
+        let response
+
+        response = await request(app)
+          .delete(DEFAULT_ROUTE + idCardToDelete)
+          .set("Authorization", AUTHORIZATION_KEY)
+        expect(response.status).toBe(204)
```

Comment:
**nitpick:** Yea, that's totally enough - you don't have to check it by GETin this card later